www/js/diary/cards/TripCard.tsx

```diff
@@ -89,7 +91,7 @@ const TripCard = ({ trip }) => {
               <Text style={{fontWeight: 'bold', textDecorationLine: 'underline'}}>{trip.display_date}</Text>
             </Text>
             <Text style={{fontSize: 13, textAlign: 'center'}}>
-              {t('diary.distance-in-time', {distance: trip.display_distance, distsuffix: trip.display_distance_suffix, time: trip.display_time})}
+              {t('diary.distance-in-time', {distance: getFormattedDistance(trip.distance), distsuffix: distanceSuffix, time: trip.display_time})}
```

Comment:
They can be memoized with the `useMemo()` hook

Longer explanation in the other comment - https://github.com/e-mission/e-mission-phone/pull/1000/files#r1256581348

They aren't currently memoized - if we take this approach then definitely should be

Reply:
Yes, as I said there, we should memoize. Do you have a sense of how long it will take? I don't want to push off the RC too long, and if the increased performance of FlashList makes up for the decreased performance due to recomputation, I am fine with even pushing this to production as long as we plan to fix it in the point release.

Reply:
Performance is ok without memoization on these. I think it can wait until the next round, likely when I rewrite the rest of diary/services.js

Reply:
Leaving this open until we file an issue tracking the remaining cleanup so we don't forget it.

Reply:
Filed new issue